.travis.yml

```diff
@@ -13,6 +13,8 @@ before_install:
   - mkdir "$ANDROID_HOME/licenses" || true
   - echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55" > "$ANDROID_HOME/licenses/android-sdk-license"
   - echo -e "\n84831b9409646a918e30573bab4c9c91346d8abd" > "$ANDROID_HOME/licenses/android-sdk-preview-license"
+script:
+  - ./gradlew clean check
```

Comment:
Suggestion: You don't really need the clean, because travis is running from zero.
Beware that `check` will only run your unit tests and lint. If you want to compile the code and the APK you need the `assemble` task. Better yet, the `build` task will do both things for you, so you may only want to put this one.

Reply:
Changed :)